rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/BatchOperationResult.java

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
_Suggestions?_

Maybe only return the component delete failures to the user?
We could reuse `RestExceptionHandler.java` to handle this scenario.

1. `public ErrorMessage(String message, HttpStatus httpStatus)` (overload the constructor)
2.  `results.add(new ErrorMessage("Could not delete component id...", HttpStatus.CONFLICT));`

What do you think about this?

Reply:
Do you have a specific reason for that ? 
I would prefer that the succeeded deletions are also in the response. I think it's not so much overhead that it's necessary to prune the response. 

Maybe we can discuss this in the round during the next sw360 meeting.

Reply:
No I don't have a specific reason for that.
Of course you are absolutely right here and it was just a suggestions.
The idea was to reuse some existing class for that but it would be better to have a complete result with all entries in the response.